sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOverrides.scala

```diff
@@ -2894,11 +2894,11 @@ object GpuOverrides {
         TypeSig.all),
       (mapPy, conf, p, r) => new GpuMapInPandasExecMeta(mapPy, conf, p, r)),
     exec[FlatMapGroupsInPandasExec](
-      "The backend for Grouped Map Pandas UDF, it runs on CPU itself now but supports " +
-        " scheduling GPU resources for the Python process when enabled",
-      ExecChecks.hiddenHack(),
-      (flatPy, conf, p, r) => new GpuFlatMapGroupsInPandasExecMeta(flatPy, conf, p, r))
-        .disabledByDefault("Performance is not ideal now"),
+      "The backend for Flat Map Groups Pandas UDF, Accelerates the data transfer between the" +
+        " Java process and the Python process. It also supports scheduling GPU resources" +
+        " for the Python process when enabled.",
+      ExecChecks(TypeSig.commonCudfTypes, TypeSig.all),
```

Comment:
Yes, there is [the result](https://docs.google.com/spreadsheets/d/1xO_-YalLsl1KIgEdksmBR4UEdmpk5SVNhkFNAuhmLDI/edit#gid=858645401).
![image](https://user-images.githubusercontent.com/7280411/115332961-692e3100-a1cb-11eb-956e-77f843bba177.png)

However I am getting OOM issue for the 34G data, so there is only the result for 17G data.
```
21/04/20 03:14:01 WARN TaskSetManager: Lost task 160.0 in stage 2.0 (TID 126, 10.19.183.78, executor 0): java.lang.OutOfMemoryError: Could not allocate native memory: std::bad_alloc: RMM failure at:/home/liangcail/miniconda3/include/rmm/mr/device/detail/arena.hpp:382: Maximum pool size exceeded

```

Reply:
The numbers look good. Hopefully #2000 will take care of the OOM issues. I'll leave this comment not resolved so the numbers are clearer.

Reply:
Our columnar pipeline is using the `BatchQueue`  instead of this `HybridRowQueue` who is for rows . I am afraid the OOM will not be handled by [#2000](https://github.com/NVIDIA/spark-rapids/pull/2000) .

Reply:
Good news is the benchmark tests can pass without OOM with the PR https://github.com/NVIDIA/spark-rapids/pull/2239